test/spec/features/modeling/behavior/CreateParticipantBehaviorSpec.js

```diff
@@ -536,4 +540,40 @@ describe('features/modeling - create participant', function() {
 
   });
 
+
+  describe('copy and paste', function() {
+
+    var collaborationDiagramXML =
+      require('../../../../fixtures/bpmn/collaboration/collaboration-participant.bpmn');
+
+    beforeEach(bootstrapModeler(collaborationDiagramXML, { modules: testModules }));
+
+
+    it('should copy process ref', inject(function(canvas, copyPaste, elementRegistry) {
+
+      // given
+      var participant = elementRegistry.get('_Participant_2'),
+          rootElement = canvas.getRootElement();
+
+      copyPaste.copy(participant);
+
+      // when
+      var newElements = copyPaste.paste({
+        element: rootElement,
+        point: {
+          x: 1000,
+          y: 1000
+        }
+      });
+
+      // then
+      var newParticipant = newElements[ 0 ];
+
+      expect(is(newParticipant, 'bpmn:Participant')).to.be.true;
+
+      expect(getBusinessObject(newParticipant).get('processRef')).not.to.equal(getBusinessObject(participant).get('processRef'));
```

Comment:
Should we assert what is being copied here? (Children, extension elements).

Or do we assert this in a different place (i.e. in `modelUtil` specs).